websites/H/Home Assistant/presence.ts

```diff
@@ -0,0 +1,74 @@
+import { ActivityType, Assets } from 'premid'
+
+const presence = new Presence({
+  clientId: '1142916495991652422',
+})
+const startTimestamp = Math.floor(Date.now() / 1000)
+
+enum ActivityAssets {
+  Logo = 'https://pub-d896e4ce4d21475fab8e7d1c02eded98.r2.dev/home-assistant-logomark.png', // TODO Change to a RCD logo, along with metadata.json logo
```

Comment:
```suggestion
  Logo = 'https://pub-d896e4ce4d21475fab8e7d1c02eded98.r2.dev/home-assistant-logomark.png',
```
This will happen automatically after your PR is merged